int8 num_sectors

---
bool result
